documentation/modules/ROOT/pages/payloads.adoc

```diff
@@ -56,6 +56,47 @@ applied. The other way round they can also be used to describe a validation algo
 * Operations defined in the Aspect Model are not subject to serialization.
 * Events defined in the Aspect Model are not subject to serialization.
 
+== Example
+=== Aspect Model
+[source,turtle,subs="attributes+,+quotes"]
+----
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
```

Comment:
Please use the prefix variables in the same way, they are referred to in [Using Namespaces](https://eclipse-esmf.github.io/samm-specification/2.1.0/modeling-guidelines.html#using-namespaces):

```asciidoc
[source,turtle,subs="attributes+,+quotes"]
----
@prefix : <urn:samm:{example-ns}.myapplication:1.0.0#> .
@prefix samm: <{samm}> .
@prefix samm-c: <{samm-c}> .
@prefix samm-e: <{samm-e}> .
@prefix unit: <{unit}> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
----
```